src/main/resources/application.properties

```diff
@@ -9,3 +9,4 @@ spring.jpa.hibernate.ddl-auto=update
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.messages.basename=messages
 log.file.output=application.log
+spring.main.allow-bean-definition-overriding=true
```

Comment:
Please add in the PR description a small explanation why this option is needed.

Reply:
Remove. Not needed